docs/products/postgresql/concepts/check-iops.rst

```diff
@@ -0,0 +1,108 @@
+IOPS capability of plans
+========================
+
+This article will explain what is the typical IOPS capability used for common plans in pg:
```

Comment:
Can we add a bit more description here? Like
* explaining the IOPS acronym and why one should be interested in knowing more
* instead of pg we should write Aiven for PostgreSQL